projects/jsf/src/lib/form-data-item.service.ts

```diff
@@ -70,6 +71,9 @@ export class FormDataItemService {
       case FormDataItemType.Array:
         const arrayItems = this.getItemsFromSubschema(schemaProperty, fieldValue, pathParts, isReadOnly, isHidden);
         return new ArrayDataItem(key, name, tooltip, helpText, required, pathParts, type, fieldValue, isReadOnly, isHidden, arrayItems);
+      case FormDataItemType.Template:
+        return new TemplateDataItem(key, name, tooltip, helpText, required, pathParts, type, fieldValue, isReadOnly, isHidden,
```

Comment:
When you update the Wiki, some of these properties don't make sense for templates and should be left out (isRequired, tooltip, helpText)